src/modules/core/components/TokenEditDialog/TokenEditDialog.tsx

```diff
@@ -46,28 +47,36 @@ const MSG = defineMessages({
 });
 
 interface Props {
-  addTokenFn: (address: Address) => Promise<any>;
+  updateTokens: (addresses: Address[]) => Promise<any>;
   cancel: () => void;
   close: () => void;
+  // Colony tokens
   tokens: AnyToken[];
+  // Token list from json file. Not supported on local env
+  tokensList?: AnyToken[];
+  // Colony native token addresss
+  nativeTokenAddress?: Address;
 }
 
 interface FormValues {
   tokenAddress: Address;
   description?: string;
+  tokenAddresses: Address[];
 }
 
 const validationSchema = yup.object({
-  tokenAddress: yup
-    .string()
-    .address()
-    // @todo validate against entering a duplicate address
-    .required(),
+  tokenAddress: yup.string().address(),
```

Comment:
I don't think so. We can just check/uncheck checkboxes, we don't need to put anything to input field

Reply:
Hmmm.... 

So this will not function like this once we actually wire this up, so I guess I was thinking too far ahead.

Yeah, you can leave it like this.